certified-connectors/App Power Forms/apiDefinition.swagger.json

```diff
@@ -0,0 +1,647 @@
+                    "operationId": "MetadataFormFieldTypes"
+                  }
+                },
+                "fieldConfiguration": {
+                  "type": "object",
+                  "x-ms-dynamic-schema": {
+                    "operationId": "MetadataFieldTypeSpecificSchema",
+                    "parameters": {
+                      "fieldType": {
+                        "parameter": "fieldType"
+                      }
+                    }
+                  },
+                  "x-ms-dynamic-properties": {
+                    "operationId": "MetadataFieldTypeSpecificSchema",
+                    "parameters": {
+                      "fieldType": {
+                        "parameterReference": "body/fieldType"
+                      }
+                    }
+                  }
+                }
+              }
+            }
+          }
+        ]
+      }
+    },
+    "/AddAdaptiveCard": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "x-ms-dynamic-schema": {
+                "operationId": "AddCardSchema",
+                "parameters": {
+                  "cardData": {
+                    "parameter": "card"
+                  },
+                  "cardName": {
+                    "parameter": "name"
+                  }
+                }
+              },
+              "x-ms-dynamic-properties": {
+                "operationId": "AddCardSchema",
+                "parameters": {
+                  "cardData": {
+                    "parameterReference": "body/card"
+                  },
+                  "cardName": {
+                    "parameterReference": "body/name"
+                  }
+                }
+              }
+            }
+          }
+        },
+        "summary": "Create new form from Adaptive Card",        
+        "operationId": "AddAdaptiveCard",
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "required":["name", "card"],
+              "properties": {
+                "name": {
+                  "type": "string",
+                  "title": "Card name"
+                },
+                "card": {
+                  "type": "string",
+                  "title": "Adaptive Card JSON"
+                },
+                "cardAfterSubmit": {
+                  "type": "string",
+                  "title": "Card displayed after submit"
+                }
+              }
+            }
+          }
+        ],
+        "description": "Use this action to create a form based on Adaptive Card JSON."
+      }
+    },
+    "/AddCardSchema/{cardName}": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "No need to specify response schema",
+            "schema":{}
```

Comment:
@TerhoAntila Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide a valid response schema definition for all the operations in the swagger.

Reply:
@sriyen-msft Could you please clarify this. This and all similar findings concern an internal operation and are used by x-ms-dynamic-schema definition. Surely x-ms-dynamic-schema is supported? The makers do NOT need to use parse JSON to parse any of the responses, they will be able to use dynamic content picker (please see the attached image)
![image](https://user-images.githubusercontent.com/10154416/152110037-f90753cd-e202-496b-a42b-236b3a21e55d.png)
This comment applies to all findings with similar description.

Reply:
@TerhoAntila The "AddCard" operation might receive a response with a "Success" or "Failed" message which can be defined in the schema, like "type": "string". Empty schema definitions are not allowed anymore.

Reply:
Applies the same comment to all the similar questions.

Reply:
@sriyen-msft Just made a commit resolving empty schema definitions.